### ISSUE TITLE
Make it possible to run web apps locally

### ISSUE THREAD

**Issue by fedorov** [QIICR/dcmqi#200]

**Comment by fedorov** [comment]
post-commit hook won't be of much help, since we would need to apply it on changes to master, and those are done on github, but anyway, since I realized this too late - here's what it would be:

```
$ cat .git/hooks/post-commit
#!/bin/sh
echo "Running post-commit hook"

echo "Merging doc folder from master to gh-pages..."
git checkout gh-pages
git checkout master -- doc
git commit -m "ENH: synchronize doc from master branch"
#git push
git checkout master
echo "Done."
echo "Do not forget to push gh-pages to GitHub!"
```

**Comment by fedorov** [comment]
reference from @che85 https://gist.github.com/cobyism/4730490

**Comment by fedorov** [comment]
Interesting, I was working on something else, and this came to my attention: https://circleci.com/gh/KitwareMedical/SlicerITKUltrasound/60

Basically, we can use CI builds as a way to sync branches and also do things that update web content! So simple, but it did not cross my mind until I saw it!

@thewtex - thank you! 👍  (I looked into that as I remembered I still owe you the JOSS review!)

**Comment by thewtex** [comment]
@fedorov Yes, it works out very well -- I hope the configuration file is a good reference. And thanks for putting the time into the JOSS review :-)
